python_coreml_stable_diffusion/torch2coreml.py

```diff
@@ -336,6 +336,7 @@ def patched_make_causal_mask(input_ids_shape, dtype, device, past_key_values_len
     return mask[None, None, :, :].expand(bsz, 1, tgt_len, tgt_len + past_key_values_length)
 
 modeling_clip._make_causal_mask = patched_make_causal_mask
```

Comment:
Can you add a comment to this line and the next one listing which versions of transformers require each of these?